Sources/PitchSpeller/Data Structures/Cross.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  Cross.swift
+//  PitchSpeller
+//
+//  Created by Benjamin Wetherfield on 6/30/18.
+//
+
+struct Cross<T,U>: Pairable {
```

Comment:
Can you add doc comments for everything (`///`)?

 It can be something almost trivial, but even within Xcode, the doc comments are visible when option-cliicking a variable, method, or type. Further, once we get the doc generation machine running again, it will make a huge difference.

I would insert the following `// MARK: - ` comments. See below.